packages/cli-kit/src/constants.ts

```diff
@@ -29,7 +29,8 @@ const constants = {
     // Variables to detect if the CLI is running in a cloud environment
     spin: 'SPIN',
     codespaces: 'CODESPACES',
-    gitpod: 'GITPOD_WORKSPACE_ID',
```

Comment:
there's a `webIDEPlatform` function in environment.local that we use for analytics. Would be nice to use these there so that we can track where the CLI is running. currently it only knows about CODESPACES.

Reply:
good catch, I thought we didn't have this tracked. Added gitpod 👌